### PR TITLE
SNAP-398 Adjust Highlighting & Search Behavior for ES. Changes in hig…

### PR DESCRIPTION
**Pull request by igor-chornobay-engagepoint**
…hlightning.

### Jira Story

- [SNAP-398
Adjust Highlighting & Search Behavior for ES](https://osi-cwds.atlassian.net/browse/SNAP-398)

## Description

changed query highlighting setting to be only two fields: autocomplete search bar and DOB as text.
Changed the people search selector a little bit. It already had a logic to take highlights from autocomplete search bar

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by wmitchell-cambria** [comment]
This is really difficult to review, because the change is small and the potential impact is large. Can you include some screenshots either here or on the story, so that we can see that the fields are still highlighting as expected?

**Comment by igor-chornobay-engagepoint** [comment]
image

File: app/services/person_search_query_builder.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # PeopleSearchQueryBuilder is a service class responsible for creation
 # of an elastic search person search query
 class PersonSearchQueryBuilder
-  NUMBER_OF_FRAGMENTS = 5
+  NUMBER_OF_FRAGMENTS = 10
```
Comment (wmitchell-cambria) [inline app/services/person_search_query_builder.rb]: I'm curious why this was bumped to 10. Would that have negative performance implications?

Comment (igor-chornobay-engagepoint) [reply app/services/person_search_query_builder.rb]: this has to do with how many (max) highlight fragments returned per field. As we now based most of highlighting on auto search bar I needed to bump it.

**Pull request merged**